docs/commons/spark.md

```diff
@@ -4,7 +4,7 @@
 [![Stars on DockerHub][dhstars]][dhrepo]
 
 This repository contains a Docker image of Apache Spark. Currently we support
-Spark versions 1.5.1, 2.1.0, 2.3.1 and 2.4.5. The lastest tag corresponds to version 2.4.5.
+Spark versions 1.5.1, 2.1.0, 2.3.1 and 2.4.5. The latest tag corresponds to version 2.4.5.
```

Comment:
Do we even have a `latest` tag anymore?
The plan was to remove that from everywhere.